### PR TITLE
Branch sort stall by location or rating

### PR DESCRIPTION
**Pull request by Ruizhi2001**

### CODE REVIEW

File: src/main/java/seedu/address/logic/Logic.java [hunk 1]
```diff
@@ -38,6 +38,10 @@ public interface Logic {
 
     /** Return the desired filtered item */
     Item getFilteredItem();
+
+    /** Returns an unmodifiable view of the filtered list of items */
```
Comment (teoks0199) [inline src/main/java/seedu/address/logic/Logic.java:42]: seems like this is getting the actual list itself, so it's not really unmodifiable

File: src/test/java/seedu/address/logic/commands/SortStallLocationCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for SortStallLocationCommand.
+ */
+public class SortStallLocationCommandTest {
```
Comment (teoks0199) [inline src/test/java/seedu/address/logic/commands/SortStallLocationCommandTest.java:16]: Nice to see testing 👍

**Review by teoks0199 — APPROVED** [review]
lgtm

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java
Comment (samuelmui8) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:1]: perhaps can abstract sort to be 1 command that accepts parameters to implement the correct comparator

**Review by samuelmui8 — APPROVED** [review]
lgtm

**Pull request merged**